extensions/wikia/AdEngine3/src/wad/rec-injector.js

```diff
@@ -1,6 +1,6 @@
 import { utils } from '@wikia/ad-engine';
 
-const wikiaApiController = 'AdEngine3ApiController';
+const wikiaApiController = 'AdEngine3EndpointController';
```

Comment:
Why separate pull request? We can release together with PHP changes

Reply:
@dmnsobczak said that it way cause problems.

Reply:
Yes, what if JS cache will be cleared off faster and it will start asking for api endpoint which don't exists yet in PHP? It will cause lot of 500 errors. Am I right @fraszczakszymon?

Reply:
There is no way to serve new JS files without new PHP code. Once you get new JS file you are sure that new endpoint is available on PHP side
@dmnsobczak @Bielik20